### DISCUSSION TITLE
pybricks camera support

### DISCUSSION THREAD

**Discussion by oscarmklaw** [pybricks/support#519] [General]
Will pybricks support the camera streaming? I would like to use robot inventor for AI STEM projects.

**Comment by dlech** [comment]
No, the camera will have to stream to a computer or have its own processor to process the information. Once processed, it could be converted into a command that could be sent to a hub running Pybricks.

**Reply by Gcopper1984** [reply]
Hi,

   "Once processed, it could be converted into a command that could be sent to a hub running Pybricks."

How can I send command to the hub?

**Reply by dlech** [reply]
Using the [pybricksdev](https://github.com/pybricks/pybricksdev) library. You should be able to find some examples by searching the issues and discussions here.
